src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddPropertiesToDeposit.scala

```diff
@@ -43,7 +43,8 @@ case class AddPropertiesToDeposit(row: Int, entry: (DatasetID, Dataset))(implici
 }
 object AddPropertiesToDeposit {
   // The email needs to be acquired (from LDAP) only once during the program execution
-  private var datamanagerEmailaddress: Try[String] = null
+  private var datamanagerEmailaddress: Try[String] = _
```

Comment:
As an aside and future outlook: @PaulBoon implemented it in this way to make sure the `datamanagerEmailaddress` is calculated only once. I see possibilities to implement a 'return type' in the `execute` method, such that we can do this LDAP search in a separate action. I will get to this in another PR once I finish the related R&D 😉